packages/patternfly-4/react-core/src/components/Form/examples/HorizontalForm.js

```diff
@@ -97,19 +97,13 @@ class HorizontalForm extends React.Component {
             id="horizontal-form-exp"
           />
         </FormGroup>
-        <FormGroup label="Can we follow up via email?" fieldId="horizontal-radio1">
-          <Radio label="Yes" aria-label="yes" id="horizontal-radio1" name="horizontal-radios" />
-          <Radio label="No" aria-label="no" id="horizontal-radio2" name="horizontal-radios" />
+        <FormGroup>
+          <Checkbox label="Follow up via email" id="alt-form-checkbox-1" name="alt-form-checkbox-1" />
+          <Checkbox label="Remember my password for 30 days" id="alt-form-checkbox-2" name="alt-form-checkbox-2" />
```

Comment:
Can you put each of these in their own `FormGroup`?

Reply:
Sure can.